hack/test-stirunimage.sh

```diff
@@ -8,14 +8,14 @@ export PATH="$PWD/_output/local/bin/$(go env GOHOSTOS)/$(go env GOHOSTARCH):$PAT
 
 function time_now()
 {
-  date +%s000
+    date +%s000
```

Comment:
I'm sorry, I noticed that the whitespace in the file was a bit mis-matchy and decided to pipe it through a formatter.